specificatie/verblijfplaats.yaml

```diff
@@ -8,23 +8,27 @@ components:
   schemas:
     Adresregel1:
       type: string
+      pattern: ^[a-zA-Z0-9À-ſ \.\-\']{1,40}?$
       description: |
         Het eerste deel van een adres is een combinatie van de straat en huisnummer.
       # example: "1600 Pennsylvania Avenue NW"
       example: "Laan van de landinrichtingscommissie Duiven-Westervoort 26A-3"
     Adresregel2:
       type: string
+      pattern: ^[a-zA-Z0-9À-ſ \,\.\-\']{1,50}?$
       description: |
         Het tweede deel van een adres is een combinatie van woonplaats eventueel in combinatie met de postcode.
       # example: "Washington, DC 20500"
       example: "1234AA Nootdorp"
     Adresregel3:
       type: string
+      pattern: ^[a-zA-Z0-9À-ſ \,\.\-\']{1,35}?$
```

Comment:
in deze en de andere adresregels kunnen ook andere tekens staan om een locatie in het buitenland aan te duiden. Ik denk dan aan º ( ) " + ; ~ # & / ¦

In de GBA-V proefomgeving dit soort tekens ook in buitenlandse adresregels of locatiebeschrijving

Reply:
@MelvLee wanneer we # & en ; toevoegen, houden we dan nog iets aan bescherming over door dit pattern?

Reply:
Is het terecht dat dit soort tekens in de adresregel mogen voorkomen. Als ja, dan moeten die tekens worden toegevoegd en moet er op een andere manier extra validatie worden toegevoegd. Het is niet mogelijk om met alleen patterns de api secure te maken.

Reply:
Betekent dit dat de patterns op de Adresregels verdwijnen ?

Reply:
Ik heb nu bij de adresregels alleen de maxLength opgenomen.